qa/rpc-tests/sc_big_commitment_tree_getblockmerkleroot.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+# Copyright (c) 2014 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import initialize_chain_clean, \
+     start_nodes, assert_false, mark_logs
+from test_framework.test_framework import ForkHeights, JSONRPCException
+
+NUMB_OF_NODES = 1
+DEBUG_MODE = 1
+FT_LIMIT = 4095     # Keep this aligned with CCTPlib
+
+class BigCommitmentTree_getblockmerkleroot(BitcoinTestFramework):
+
+    def setup_chain(self):
+        print("Initializing test directory " + self.options.tmpdir)
+        initialize_chain_clean(self.options.tmpdir, NUMB_OF_NODES)
+
+    def setup_network(self, split=False):
+        self.nodes = start_nodes(NUMB_OF_NODES, self.options.tmpdir,
+                                 extra_args=[['-logtimemicros=1', '-debug=sc', '-debug=py', '-debug=cert']] * NUMB_OF_NODES)
+        self.is_network_split = split
+
+    def run_test(self):
+        '''
+        1. Mine blocks until reaching the sidechain fork; initialize tx_data with valid transaction data
+        2. Create a raw tx
+        3. Generate merkleTree and scTxsCommitmentTree for (FT_LIMIT - 1), FT_LIMIT and (FT_LIMIT + 1) txs (current FT_LIMIT is 4095)
+        4. Check if the ScTxsCommitmentTree stops updating due to limited height
+        '''
+
+        # Reach SC v2 fork height
```

Comment:
I guess this comment should be `# Reach SC fork height`.

Reply:
Yes, corrected.